doc/ss_logger.rst

```diff
@@ -0,0 +1,172 @@
+Configuration
+=============
+-------------
+Log to stdout
+-------------
+.. _log_to_stdout:
+The ``get_logger()`` function in SmartSim enables users to initialize
+a logger instance. Once initialize, a user may use the instance
+to specify a log level function and include a log message.
+A user may control which messages are logged via the
+`SMARTSIM_LOG_LEVEL` which defaults to `info`.
+
+To use the SmartSim logger within the Python script, import the required module:
+
+.. code-block:: python
+
+      from smartsim.log import get_logger
+
+Next, initialize an instance of the logger and provide a `name`:
+
+.. code-block:: python
+
+      logger = get_logger("SmartSim")
+
+To demonstrate full functionality of the SmartSim logger, we include all log
+functions in the Python driver script with log messages:
+
+.. code-block:: python
+
+      logger.info("This is a message")
+      logger.debug("This is a debug message")
+      logger.error("This is an error message")
+      logger.warning("This is a warning message")
+
+Execute the script *without* setting the `SMARTSIM_LOG_LEVEL`.
+Note that `SMARTSIM_LOG_LEVEL` defaults to `info`.
+When we execute the script,
+the following messages will print to stdout::
+    11:15:00 system.host.com SmartSim[130033] INFO This is a message
+    11:15:00 system.host.com SmartSim[130033] ERROR This is an error message
+    11:15:00 system.host.com SmartSim[130033] WARNING This is a warning message
+
+Notice that the `debug` messages were filtered. This is because by using
+a higher severity level (`info`), we instruct SmartSim to omit the lower severity level (`debug`).
+
+Next, set `SMARTSIM_LOG_LEVEL` to `debug` in terminal::
+    export SMARTSIM_LOG_LEVEL=debug
+
+When we execute the script,
+the following messages will print to stdout::
+    11:15:00 system.host.com SmartSim[65385] INFO This is a message
+    11:15:00 system.host.com SmartSim[65385] DEBUG This is a debug message
+    11:15:00 system.host.com SmartSim[65385] ERROR This is an error message
+    11:15:00 system.host.com SmartSim[65385] WARNING This is a warning message
+
+Notice that all log messages printed to stdout. By using
+the lowest severity level (`debug`), we instruct SmartSim print all log levels.
+
+Next, set `SMARTSIM_LOG_LEVEL` to `quiet` in terminal::
+    export SMARTSIM_LOG_LEVEL=quiet
+
+When we run the program once again,
+the following output is printed to stdout::
+    11:15:00 system.host.com SmartSim[65385] ERROR This is an error message
+    11:15:00 system.host.com SmartSim[65385] WARNING This is a warning message
+
+Notice that the `info` and `debug` messages were filtered. This is because by using
+the highest severity level (`quiet`), we instruct SmartSim to omit the lower severity levels
+(`info` and `debug`).
+
+-----------
+Log to File
+-----------
+.. _log_to_file:
+The ``log_to_file()`` function in SmartSim allows users to log messages
+to a specified file by providing a file `name`. If the file name
```

Comment:
Please clarify whether it only accepts a filename or you can also provide a path to a file as well. Absolute or relative path?